parts/linux/cloud-init/artifacts/labels.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Update Labels for Kubernetes nodes
+After=kubelet.service
+[Service]
+Restart=always
+RestartSec=60
```

Comment:
Should we back this off to 5 minutes to be nice to very large clusters with lots of nodes?

Reply:
Updated to 5 minutes. Curious what other reviewers think.

Reply:
Changed to 30 seconds with retry on failure